Components/Motors/TalonMC.cs

```diff
@@ -94,9 +94,9 @@ public void SetSpeed(float Speed)
         private void SetSpeedDirectly(float Speed)
         {
             if (Speed > this.MaxSpeed) { Speed = this.MaxSpeed; }
-            if (Speed * -1 > this.MaxSpeed) { Speed = -1 * this.MaxSpeed; }
+            if (-Speed > this.MaxSpeed) { Speed = -this.MaxSpeed; }
             if (this.Stopped) { Speed = 0; }
-            this.PWMOut.SetOutput(Speed / 6.0f + 0.5f);
+            this.PWMOut.SetOutput((Speed + 1.0f) * (0.3333333f) / (2.0f) + 0.3333333f);
```

Comment:
Personal preference, but I'd prefer if we used fractions, because it looks less janky in my opinion